src/common/ui-wrapper.js

```diff
@@ -38,6 +38,25 @@ class UIWrapper {
     this.setConfig(Utils.Object.mergeDeep({}, previewThumbnailConfig, seekbarConfig), 'seekbar');
   }
 
+  handleVr(config: KalturaPlayerOptionsObject): void {
+    if (this._disabled) return;
+    if (Utils.Object.getPropertyPath(config, 'plugins.vr')) {
+      this._setFullscreenConfig();
+      this._setStereoConfig(config)
+    }
+  }
+
+  _setFullscreenConfig(): void {
+    const fullscreenConfig = Utils.Object.getPropertyPath(this._uiManager, 'config.components.fullscreen');
+    this.setConfig(Utils.Object.mergeDeep({}, {inBrowserFullscreenForIOS: true}, fullscreenConfig), 'fullscreen');
+  }
+
+  _setStereoConfig(config: KalturaPlayerOptionsObject): void {
+    if (config.plugins.vr.toggleStereo || (Env.device.type && config.plugins.vr.toggleStereo !== false)) {
```

Comment:
not sure I understand this condition

Reply:
@OrenMe  `toggleStereo` is `false` by default for desktop and `true` for mobile. is it clear?